awx/main/models/events.py

```diff
@@ -360,11 +360,10 @@ def _update_from_event_data(self):
             value = force_text(event_data.get(field, '')).strip()
             if value != getattr(self, field):
                 setattr(self, field, value)
-        if isinstance(self, JobEvent):
```

Comment:
@chrismeyersfsu @AlanCoding I totally broken event logging for project updates while refactoring recently 🤦‍♂